config/v1/types_cluster_version.go

```diff
@@ -225,7 +225,7 @@ type UpdateHistory struct {
 type ClusterID string
 
 // ClusterVersionCapability enumerates optional, core cluster components.
-// +kubebuilder:validation:Enum=openshift-samples;baremetal;marketplace
+// +kubebuilder:validation:Enum=openshift-samples;baremetal;marketplace;insights
```

Comment:
@deads2k When an enum like this is using the wrong casing, what are your thoughts on updating it to allow these existing values but making new values PascalCase to fit the expected Kube conventions?

Something like this maybe with a developer note as to why some are duplicated. Consumers would have to be updated to observe both old and new values

```suggestion
// ---
// + Old lowercase values left for compatibility, new values should be added as PascalCase
// +kubebuilder:validation:Enum=OpenShiftSamples;Baremetal;MarketPlace;Insights;openshift-samples;baremetal;marketplace
```

Reply:
The sins of the past.  I am in favor of addressing this and the proposed solution is reasonable, but it will require a backport and an explanation (probably better a PR) of how to the CVO will handle it and backport.  I'd like to work out and merge the CVO handling *before* adding the toleration here.

If @wking is amenable to making the CVO ToLower the value, then merging this one as "Insights" makes sense. If he's not, then I would release this one as "insights" and try to convince him before making the change.

Reply:
Renamed it to `Insights`